.github/workflows/cmake.yml

```diff
@@ -18,12 +19,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-latest, ubuntu-latest, windows-latest]
```

Comment:
`ubuntu-latest` uses 24.04, which may be too new.
Using an older distro here so that glibc won't be too new for most users.

Reply:
This file is only for compile-time and (hopefully at some point) runtime tests.

Make a new Linux-only build script based on `cmake.yml`. Name it something like "ubuntu-package". Make it run only on tags and only on your preferred old-but-not-EOL LTS branch.